server/utils/licenceStatus.js

```diff
@@ -163,6 +163,26 @@ function getEligibilityStageState(licence) {
     };
 }
 
+function isEligible({
+    excluded,
+    insufficientTime,
+    unsuitable,
+    exceptionalCircumstances
+}) {
+    if (excluded) {
+        return false;
+    }
+    if (insufficientTime) {
+        return false;
+    }
+
+    if (unsuitable && !exceptionalCircumstances) {
+        return false;
+    }
+
+    return true;
+}
```

Comment:
Is this properly tested?

Reply:
My assumption here is that it's just a longer version of `!(excluded || insufficientTime || unsuitable)`
And if the the tests still pass for those then I guess it is?

Reply:
Add some test coverage